index.bs

```diff
@@ -2246,16 +2282,19 @@ authenticator.
 ## Extending request parameters ## {#extension-request-parameters}
 
 An extension defines up to two request arguments. The <dfn>client argument</dfn> is passed from the [=[RP]=] to the client
-in the {{getAssertion()}} or {{makeCredential()}} call, while the <dfn>authenticator argument</dfn> is passed from the client
+in the {{CredentialsContainer/get()}} or {{create()}} call, while the <dfn>authenticator argument</dfn> is passed from the client
 to the authenticator during the processing of these calls.
 
 A [RP] simultaneously requests the use of an extension and sets its client argument by including an entry in the
-{{ScopedCredentialOptions/extensions}} option to the {{makeCredential()}} or {{getAssertion()}} call. The entry key MUST be the
+{{ScopedCredentialOptions/extensions}} option to the {{create()}} or {{CredentialsContainer/get()}} call. The entry key MUST be the
 extension identifier, and the value MUST be the [=client argument=].
 
 <pre class="example" highlight="js">
-    var assertionPromise = credentials.getAssertion(..., /* extensions */ {
-        "webauthnExample_foobar": 42
+    var assertionPromise = navigator.credentials.get({
```

Comment:
Taken care of in https://github.com/w3c/webauthn/pull/384/commits/c48c70bc9ca6183e4e473d4c02fcc5cbd6a6562a.